frontend/src/components/SiteHeader/SiteHeader.tsx

```diff
@@ -85,62 +90,81 @@ function SiteHeader() {
+                    right: 0,
+                    bottom: 0,
+                    background: 'rgba(0, 0, 0, 0.1)',
+                    zIndex: 1000,
+                    display: sidebarOpen ? 'block' : 'none'
+                }}
+                onClick={closeSidebar}
+            />
+            <div ref={sidebarRef} style={{
+                position: 'fixed',
+                top: 0,
+                left: 0,
+                width: '220px',
+                height: '100%',
+                background: 'linear-gradient(347deg in oklab, rgb(0% 92% 99% / 100%) -15% -15%, rgb(84% 0% 55% / 100%) 132% 132%)',
+                transform: sidebarOpen ? 'translateX(0)' : 'translateX(-100%)',
+                transition: 'transform 0.3s ease-in-out',
+                zIndex: 1001,
+                padding: '20px 30px',
             }}>
-                <Link style={{color: '#fff9e6', textDecoration: 'none', fontSize: '20px'}} to="/offers">offers</Link>
-                <Link style={{color: '#fff9e6', textDecoration: 'none', fontSize: '20px'}} to="/asks">asks</Link>
-                <Link style={{color: '#fff9e6', textDecoration: 'none', fontSize: '20px'}} to="/profile">profile</Link>
-                <Link style={{color: '#fff9e6', textDecoration: 'none', fontSize: '20px'}} to="/supportus">about</Link>
-                <SignedIn>
-                    <SignOutButton>
-                         <LogOut size={20} />
-                        
-                    </SignOutButton>
-                </SignedIn>
-                <SignedOut>
-                    <SignInButton>
-                        <button style={{
-                            background: 'linear-gradient(to left, #1a1a66, #2a2a88)',
-                            color: '#fff9e6',
-                            border: '1px solid #fff9e6',
-                            borderRadius: '5px',
-                            padding: '10px 20px',
-                            cursor: 'pointer',
-                            fontSize: '16px',
-                            textShadow: '0 0 6px #fff9e6'
-                        }}>Sign In</button>
-                    </SignInButton>
-                </SignedOut>
-            </nav>
-        </div>
+                <button onClick={closeSidebar} style={{
+                    position: 'absolute',
+                    top: '20px',
+                    right: '10px',
+                    background: 'none',
+                    border: 'none',
+                    color: '#fff9e6',
+                    fontSize: '24px',
+                    cursor: 'pointer'
+                }}>
+                    <ArrowLeftToLine size={24} />
+                </button>
+                <nav style={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    gap: '20px',
+                    marginTop: '50px'
+                }}>
+                    <Link to="/profile" onClick={() => setSidebarOpen(false)}>
```

Comment:
<Avatar> itself already has a <Link> component in it. Using this throws an error in the console.